docs/plugin_developers_guide/09_packaging.rst

```diff
@@ -0,0 +1,256 @@
+.. Documentation of package for a Plugin
+
+Packaging Documentation
+=======================
+
+Below are the prerequisite files which we need them in Plugin repository itself.
```

Comment:
There are as set of files that must exist within the repository of a plugin that are used to create the package for that plugin on the various supported platforms.  The following documents what those files are and what they should contain.